core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -29,6 +29,16 @@ private[spark] object Status {
     .timeConf(TimeUnit.NANOSECONDS)
     .createWithDefaultString("100ms")
 
+  val LIVE_ENTITY_UPDATE_STALENESS_LIMIT = ConfigBuilder("spark.ui.liveUpdate.stalenessLimit")
+    .internal()
```

Comment:
Why internal? Spark doesn't set it itself. If anyone is going to change it, it will be users.